build.gradle

```diff
@@ -237,6 +235,7 @@ dependencies {
     implementation "com.icthh.xm.commons:xm-commons-topic:${xm_commons_version}"
     implementation "com.icthh.xm.commons:xm-commons-logging-configurable:${xm_commons_version}"
     implementation "com.icthh.xm.commons:xm-commons-logging-trace:${xm_commons_version}"
+    implementation "com.icthh.xm.commons:xm-commons-client-feign:${xm_commons_version}"
```

Comment:
please add to readme how to use feign client. 
Even if it is fully automated configuration it worth to add mention that you need to add `xm-commons-client-feign` dependency.